engine/src/main/java/org/terasology/engine/world/generator/internal/WorldGeneratorManager.java

```diff
@@ -39,34 +40,32 @@ public WorldGeneratorManager(Context context) {
     }
 
     public void refresh() {
-        ModuleManager moduleManager = context.get(ModuleManager.class);
```

Comment:
Reviewers note: this diff is much easier to read on GitHub if you ⚙️ ☑️  _Hide whitespace changes_